SSA/Core/WellTypedFramework.lean

```diff
@@ -684,3 +686,13 @@ register_simp_attr TypedUserSemantics.argUserType
 register_simp_attr TypedUserSemantics.outUserType
 register_simp_attr TypedUserSemantics.regionDom
 register_simp_attr TypedUserSemantics.regionCod
+
+
+structure SSA.Executable [Goedel β] where
+  (input : SSA.UserType β)
+  (output : SSA.UserType β)
```

Comment:
Maybe call these `inputType` and `outputType`. My first thought is that these fields might be input and output data that are fed to / produced by some executable